frisby.go

```diff
@@ -208,9 +210,11 @@ func (F *Frisby) Send() *Frisby {
 	if Global.PrintProgressName {
 		fmt.Println(F.Name)
 	} else if Global.PrintProgressDot {
-		fmt.Printf(".")
+		fmt.Printf("")
```

Comment:
There is no progress dot in the output of _go test_. go-junit-report would fail having a dot at the beginning of each line.